src/TerminalFighter/Missile/Missile.cpp

```diff
@@ -33,8 +33,24 @@ void Missile::draw(I_GraphicsHandler& graphics) {
         set_texture(graphics.load_image(texture_path_));
     }
 
-    graphics.draw(missile_texture_, (int)x_pos(), (int)y_pos(), GraphicPriority::MIDDLE, true,
-                  util::angle(0, 0, x_vel_, y_vel_), NULL);
+    if (!spriteAnimator_.is_initialized()) {
+        //3 columns, 2 rows of sprite sheets, update every 6 frames
+        spriteAnimator_.init(missile_texture_, 2, 3, 3);
+    }
+
+
+    SDL_Rect dest_rect;
+    dest_rect.x = (int)x_pos();
+    dest_rect.y = (int)y_pos();
+    dest_rect.w = 172;
+    dest_rect.h = 30;
```

Comment:
Can we call the helper function here of just passing in the x and y pos.

Reply:
I could create another helper function here, but this would be different compared to the other functions.

The other functions automatically took the input SDL_Texture, and re-used their original `w` and `h`. In this case, we don't want to re-use it because it'd be the size of the entire sprite sheet's `w` and `h`. So we have to specify the `w` and `h` by ourselves.